tools/yaml-templates/web-e2e-tests-job.yml

```diff
@@ -0,0 +1,73 @@
+# This template handles running E2E tests (including all necessary setup steps) from the host SDK repo for various environments.
+# It will run one E2E job for each testPrefixPattern selector in the testPrefixPatternGroups parameter.
+
+parameters:
+  - name: AppHostingSdk
+    default: none
+    type: string
+  - name: hostingEnvironmentType
+    type: string
+    values:
+      - 'electron'
+      - 'standardWeb'
+  - name: teamsJsReferenceType
+    type: string
+    values:
+      - 'npm'
+      - 'scriptTag'
+  - name: testPrefixPatternGroups
+    type: object
```

Comment:
>object [](http://example.com/codeflow?start=10&length=6)

YAML doesn't have arrays as types but objects can be treated like you would expect arrays to be treated, as you can observe by looking at how this value is passed in as well as how it's iterated over below.